### DISCUSSION TITLE
appKey  appSecret

### DISCUSSION THREAD

**Discussion by alma554** [z-master42/solarflow#36] [General]
Ich versuche über den von Ihnen angegebenen curl Befehl den appKey und app Secret abzurufen. Was mir einfach nicht gelingen will. Ich bekomme immer die Meldung die auf dem Bild zu sehen ist. Können Sie mir sagen was die Meldung zu bedeuten hat bzw. was ich falsch mache.
![Screenshot 2024-09-15 111128](https://github.com/user-attachments/assets/29b0dfa6-6a98-4232-a3de-cb57a23003cd)

**Comment by z-master42** [comment]
Das könnte an einem "Problem" in deinem Netzwerk liegen, wenn es denn immer noch nicht funktioniert. Router oder Firewall o. ä.

Versuche mal als weitere Option `--ssl-no-revoke` oder `--ssl-revoke-best-effort`.

**Reply by alma554** [reply]
Danke für die Antwort. Funktionieren tut es aber auch nicht. Kann auch nichts finden was ich im Router oder der Firewall einstellen könnte.

**Comment by Hans80939** [comment]
Hallo,

„Die erste und wichtigste Änderung, für alle die seit ca. einer Woche keine Daten über die Europäischen MQTT Server von Zendure mehr bekommen haben. Die Server-Adresse hat sich geändert. Aus mqtt.zen-iot.com:1883 wurde mqtt-eu.zen-iot.com:1883.
Diese Adresse könnt ihr einfach in eurer zendure.conf ändern, danach das MQTT Add-On einmal neu starten und sofort sollten wieder werte ankommen.“

Quelle:
https://www.steuerdeinleben.de/zen-mqtt-neu/#google_vignette

**Reply by z-master42** [reply]
Ist das jetzt ein Zirkelschluss?

**Reply by Hans80939** [reply]
Sorry, ich wollte dem user einfach schnell helfen, weil es mir genauso ging und ich dann bei Thomas die Lösung gefunden habe.  Einen Zirkelbezug wollte ich nicht erstellen! Deshalb habe ich auch die Quelle angegeben.

Aber sonst ist hier alles saugut beschrieben. Es laufen zwar bei mir auch noch keine Werte sauber, aber ich denke, das liegt an der noch fehlenden Automatisierung, wie auch hier beschrieben.

**Reply by z-master42** [reply]
Alles gut. Der User ist aber noch gar nicht an der Stelle. Zudem liefert der entsprechende Befehl die richtige mqtt-Adresse mit. Der Beitrag von Thomas ist von September 2024, das Thema ist also nicht unbedingt neu. Und Zirkelschluss, weil sein Beitrag als Quelle auch wieder hierher führt.

**Comment by Hans80939** [comment]
Ja, das stimmt z-master42. Über ihn bin ich auch auf Dich gestoßen. Deine Beschreibung und Implementierung gefallen mir aber sehr gut! Das hast Du sehr gut gemacht. Danke. Ich wünsche Dir einen guten Rutsch und ein gesundes und produktives Neues Jahr 2025. Beste Grüße Hans

 

Von: z-master42 ***@***.***> 
Gesendet: Dienstag, 31. Dezember 2024 09:44
An: z-master42/solarflow ***@***.***>
Cc: Hans Hohenner ***@***.***>; Comment ***@***.***>
Betreff: Re: [z-master42/solarflow] appKey appSecret (Discussion #36)

 

Alles gut. Der User ist aber noch gar nicht an der Stelle. Zudem liefert der entsprechende Befehl die richtige mqtt-Adresse mit. Der Beitrag von Thomas ist von September 2024, das Thema ist also nicht unbedingt neu. Und Zirkelschluss, weil sein Beitrag als Quelle auch wieder hierher führt.

**Reply by z-master42** [reply]
Danke und gleichfalls. Leider ist die Sache damit, dass Zendure keine 0-Werte übermittelt ein ungelöstes Problem, da auch die Automatisierungen, um die Entitäten auf 0 W zu setzen eine Krücke sind.

**Reply by Hans80939** [reply]
Geb Dir recht! Ist das mit der solitären MQTT-Installation von Zendure auf einem HA-System dann das gleiche? Falls ja, dann sollten wir mal bei Zendure Druck machen, dass die Ihre MQTT-Ausgabe definiert machen und somit richtig stellen.

**Reply by z-master42** [reply]
Was meinst du mit solitärer Installation? Solange diese beiden Zendure Broker im Spiel sind, ist es das gleiche.

Bzgl. der Übermittlung ist das schon lange bei Zendure offen und es war ganz zu Beginn auch tatsächlich mal anders, da hat es nämlich sauber funktioniert mit den 0 W.

Siehe https://github.com/Zendure/developer-device-data-report/issues/47

**Comment by Hans80939** [comment]
Ich meine mit solitär die alleinige Übertragung zwischen Zendure und HA ohne eine Bridge.

**Reply by z-master42** [reply]
Macht keinen Unterschied, da es nicht an der Bridge liegt, sondern an den Zendure Brokern.
